recipes/cassiopee/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: cassiopee
+  version: "1.0.5"
+
+build:
+  number: 0
+
+source:
+  fn: 1.0.5.tar.gz
+  url: https://github.com/osallou/cassiopee-c/archive/1.0.5.tar.gz
+  md5: 38f07e5d2ee2a55679281bce73f9993e
+
+requirements:
+  build:
+    - glog
+    - cppunit
+    - boost
```

Comment:
please pin the boost dependency as described here: https://bioconda.github.io/guidelines.html#c-c